ethcore/src/spec/spec.rs

```diff
@@ -220,6 +223,10 @@ impl From<ethjson::spec::Params> for CommonParams {
 				BlockNumber::max_value(),
 				Into::into,
 			),
+			eip145_transition: p.eip145_transition.map_or(
+				BlockNumber::max_value(),
+				Into::into,
+			),
```

Comment:
not an issue, but wouldn't it may be better to write:

```rust
			eip145_transition: p.eip145_transition.map_or_else(
				BlockNumber::max_value,
				Into::into,
			),
```

Reply:
I changed all instance of this in `spec.rs` to use `map_or_else`.

Reply:
@debris what is the reason for preferring `map_or_else` over `map_or`? Is it because the arguments to `map_or` are eagerly evaluated?

Reply:
I think so, yeah. Although `BlockNumber` is just `u64`, so `max_value()` most likely inlined by the compiler anyway.
I like the change though, because of consistency across the codebase.